### PR TITLE
Add pageview to an array of pages

### LINKED ISSUE #8841: Add Pageview to more pages
As part of new tracking for Search, we want to add Pageview information to

- All new search pages
- All stories/events/exhibitions pages
- Concepts(?)

We need to determine which `properties` are useful to pass in and decide on a `name` for each of the above.
```
pageview: {
   name: 'search',
   properties: [TBD]
}
```

### PR DESCRIPTION
**Pull request by rcantin-w**
## Who is this for?
Analytics

## What is it doing for them?
Tracks more pages:

```
- /search
- /search/images
- /search/works
- /search/events
- /search/exhibitions
- /search/stories

- /events/[id]
- /exhibitions/[id]

- /series/[id]
- /articles/[id]
- /books/[id]

- /concepts/[id]
```

Some of the properties passed are subject to change.

Closes #8841

### CODE REVIEW

File: catalogue/webapp/pages/search/events.tsx [hunk 1]
```diff
@@ -73,6 +76,13 @@ export const getServerSideProps: GetServerSideProps<
     props: removeUndefinedProps({
       serverData,
       eventResponseList,
+      pageview: {
+        name: 'exhibitions',
```
Comment (harrisonpim) [inline catalogue/webapp/pages/search/events.tsx:80]: should this be an 'event'?

File: content/webapp/pages/article-series.tsx [hunk 2]
```diff
@@ -117,6 +119,10 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
         gaDimensions: {
           partOf: series.seasons.map(season => season.id),
         },
+        pageview: {
+          name: 'stories',
```
Comment (rcantin-w) [inline content/webapp/pages/article-series.tsx:123]: @harrisonpim should this be singular?

File: catalogue/webapp/pages/search/events.tsx (hunk 1 above)
Comment (rcantin-w) [reply catalogue/webapp/pages/search/events.tsx]: Good catch thanks!

File: content/webapp/pages/article-series.tsx (hunk 2 above)
Comment (harrisonpim) [reply content/webapp/pages/article-series.tsx:123]: yeah, if we're following the existing pattern for works and images then we should use `stories` for the search results, and `story` for the thing in the list of results which gets clicked on - i think that makes this (and some of the others) a `story`

**Review by harrisonpim — APPROVED** [review]
✨ nice!

**Review by alexwlchan — APPROVED** [review]

**Pull request merged**